trunk/SUAVE/Methods/Weights/Buildups/Electric_Vectored_Thrust/empty.py

```diff
@@ -74,16 +65,16 @@ def empty(config,
 # Assumed Weights
 #-------------------------------------------------------------------------------
 
-    output.payload          = config.propulsors.network.payload.mass_properties.mass
+    output.payload          = config.propulsors.propulsor.payload.mass_properties.mass
     output.seats            = 30.
     output.avionics         = 15.
-    output.motors           = 10 * config.propulsors.network.number_of_engines
-    output.battery          = config.propulsors.network.battery.mass_properties.mass
-    output.servos           = 0.65 * config.propulsors.network.number_of_engines
+    output.motors           = 10 * config.propulsors.propulsor.number_of_engines
+    output.battery          = config.propulsors.propulsor.battery.mass_properties.mass
+    output.servos           = 0.65 * config.propulsors.propulsor.number_of_engines
     output.rotor_servos     = 2 * (len(config.wings['main_wing'].motor_spanwise_locations) 
-                                   + len(config.wings['main_wing'].motor_spanwise_locations))
+                                   + len(config.wings['main_wing_2'].motor_spanwise_locations))
```

Comment:
Is this a required tag? We should find a different way to handle this.

Reply:
Changed to handle a generic number of wings of  any name

Reply:
I don't see changes here.

Reply:
I corrected similar syntax at the bottom of the script but forgot this one. Thanks. changes made